packages/ndla-ui/src/model/ContentType.ts

```diff
@@ -17,6 +17,7 @@ export const MULTIDISCIPLINARY_TOPIC = "multidisciplinary-topic";
 export const CONCEPT = "concept";
 export const EXTERNAL = "external";
 export const MISSING = "missing";
+export const CONCEPT_ARTICLE = "conceptArticle";
```

Comment:
Burde denne være kebab-case som de andre er?

Reply:
Bortsett fra missing og external tilsvarer alle disse parent resourceTypes. CONCEPT rett over var lagt inn for å dekke dette behovet, men ser jo at det kolliderer med feltet i resourceEmbedTypeMapping